Birthday-2024-Project/MainScenes/cam8E62.tmp

```diff
@@ -0,0 +1,22 @@
+[gd_scene load_steps=4 format=3 uid="uid://bvxs6val4gfwt"]
```

Comment:
is this file used anywhere?

Reply:
I don't think so? It was added in your commit labeled "experimental"
https://github.com/Saplings-Projects/Birthday-2024/pull/55/commits/593ac5559259ab5c05cfa5f69143da5fcee25af8

Reply:
wtf, so it must be a garbage, we can remove it

Reply:
removed